haskell/src/Stablecoin/Client/Cleveland/StablecoinImpl.hs

```diff
@@ -8,13 +8,12 @@ module Stablecoin.Client.Cleveland.StablecoinImpl
 
```

Comment:
:thinking:  Do we still need the `StablecoinImpl` type?

At a quick look I think that it might be replaced with stuff that's now part of the `morley-client` library, but I may be forgetting something.

BTW even if we can, it looks like it might be a heavy and not-so-necessary-yet update to make, so I'd be ok to leave it for a follow-up PR.

Reply:
`StablecoinImpl` uses `stablecoin-client` to run operations instead of RPC or tezos-client. The point I guess is to test the binary. It's debatable how useful that is exactly, and whether this could be achieved by a simpler (less hacky) approach, but that certainly would be out of scope here, and I don't know how much additional time we're willing to spend on it (AFAIU the project is in the maintenance mode)

Reply:
> The point I guess is to test the binary.

I suspect that the reason is (or was) lack of choice. This project is old enough that I believe the functionalities needed weren't in `morley-client` yet (or perhaps `morley-client` wasn't a thing at all), but I'm not sure.

> AFAIU the project is in the maintenance mode

That's true, but the update is still a sensible thing to do to reduce the maintenance cost down the line.

Although there is no need to do this now, nor we should stop this PR for this IMO.

Also I just remembered that we have #182 for this specifically :sweat_smile: